compiler/src/dotty/tools/AbstractFileClassLoader.scala

```diff
@@ -11,7 +11,6 @@
  */
 
 package dotty.tools
-package repl
```

Comment:
Let's move this to `dotty.tools.io`, rather than top level dotty.tools.

Reply:
I was impressed that `AFCL` finally got out deeply nested `util` package. It is a work horse. Maybe it deserves a better name under `dotty.tools`.

Reply:
I just moved it to `dotty.tools.io`.